masakari-controller/controller/masakari_controller.py

```diff
@@ -47,10 +46,14 @@
 import controller.masakari_starter as starter
 from controller.masakari_util import RecoveryControllerUtil as util
 from controller.masakari_util import RecoveryControllerUtilDb as util_db
+from controller.masakari_util import LogProcessBeginAndEnd
+from oslo_log import log as oslo_logging
 import controller.masakari_config as config
 import controller.masakari_worker as worker
 import db.api as dbapi
 
+log_process_begin_and_end = LogProcessBeginAndEnd()
+LOG = oslo_logging.getLogger('controller.masakari_controller')
 
```

Comment:
\__name__ is "\__main__" in masakari_controller.py .
I want to get logger by "controller.*" for applying default log level.
My another idea is to set "controller.masakari_controller" to \__name__.
If you have other cool ideas, please talk to me.
(\__file__ does not contain "controller".)